OmniKitUI/ViewControllers/CommandResponseViewController.swift

```diff
@@ -43,12 +43,56 @@ extension CommandResponseViewController {
             return LocalizedString("Changing time…", comment: "Progress message for changing pod time.")
         }
     }
+    
+    private static func podStatusString(status: DetailedStatus) -> String {
+        var result, str: String
+
+        let formatter = DateComponentsFormatter()
+        formatter.unitsStyle = .full
+        formatter.allowedUnits = [.day, .hour, .minute]
+        if let timeStr = formatter.string(from: status.timeActive) {
+            str = timeStr
+        } else {
+            str = String(format: LocalizedString("%1$@ minutes", comment: "The format string for minutes (1: number of minutes string)"), String(describing: Int(status.timeActive / 60)))
+        }
+        result = String(format: LocalizedString("Pod Active: %1$@\n", comment: "The format string for Pod Active: (1: Pod active time string)"), str)
+
+        result += String(format: LocalizedString("Delivery Status: %1$@\n", comment: "The format string for Delivery Status: (1: delivery status string)"), String(describing: status.deliveryStatus))
+
+        result += String(format: LocalizedString("Total Insulin Delivered: %1$@ U\n", comment: "The format string for Total Insulin Delivered: (1: total insulin delivered string)"), status.totalInsulinDelivered.twoDecimals)
+
+        result += String(format: LocalizedString("Reservoir Level: %1$@ U\n", comment: "The format string for Reservoir Level: (1: reservoir level string)"), status.reservoirLevel?.twoDecimals ?? "50+")
+
+        result += String(format: LocalizedString("Last Bolus Not Delivered: %1$@ U\n", comment: "The format string for Last Bolus Not Delivered: (1: bolus not delivered string)"), status.insulinNotDelivered.twoDecimals)
+
+        result += String(format: LocalizedString("Alerts: %1$@\n", comment: "The format string for Alerts: (1: the alerts string)"), String(describing: status.unacknowledgedAlerts))
```

Comment:
This will display unusable names like "slot4" instead of more usable names such as "lowReservoirAlarm" for each unacknowledged alert. Can the AlertSet description function be updated to return more reasonable names besides just "slotN" for each alert in an AlertSet?